sql-statements/sql-statement-show-character-set.md

```diff
@@ -11,17 +11,9 @@ This statement provides a static list of available character sets in TiDB. The o
 
 ## Synopsis
 
-**ShowStmt:**
+**ShowCharsetStmt:**
 
-![ShowStmt](/media/sqlgram/ShowStmt.png)
-
-**ShowTargetFilterable:**
-
-![ShowTargetFilterable](/media/sqlgram/ShowTargetFilterable.png)
-
-**CharsetKw:**
-
-![CharsetKw](/media/sqlgram/CharsetKw.png)
```

Comment:
These three lines should not be deleted according to https://github.com/pingcap/docs-cn/pull/3294/files.